### ISSUE TITLE
Installation in wrong directory?

### ISSUE THREAD

**Issue by JensHoRi** [jbouwh/omnikdataloggerproxy#4]
I just tried to install with sudo pip3 install omnikdataloggerproxy.

I was surprised the scripts hat been installed in /usr/local instead of /usr/local/share/omnikdataloggerproxy

Don't know how to change

**Comment by jbouwh** [comment]
Python scripts are installed in `/usr/local/bin/omnikdataloggerproxy` some other helper files might be installed in the share folder.

**Comment by JensHoRi** [comment]
O.k, found it. 

Only some examples for config and copies of other files had gone to /usr/local directly. 

Searched on the wiki but didn’t found: how to configure to send to mqtt only without another host?

**Comment by jbouwh** [comment]
> send to mqtt only without another host?

You can install omnikdatalogger via hacs, it can publish to Mqtt.

**Comment by JensHoRi** [comment]
Well, I installed it, but throws out errors. Maybe because I have deye 300 using another Protocol?

**Comment by JensHoRi** [comment]
Please close this, will open separate one

**Comment by jbouwh** [comment]
Well the older inverters might need omnikdataloggeproxy because they cannot be read out directly. The newer ones shoul not need it. What year are your inverters installed?

**Comment by JensHoRi** [comment]
One last year, two this year.

**Comment by jbouwh** [comment]
In that case try the tcpclient of omnikdatalogger

**Comment by JensHoRi** [comment]
I tried today.

First got "WARNING:omnik.datalogger:W 2023-01-29T13:52:31.506441 Inverter access over HTTP is not supported!. 401 Client Error: Unauthorized for url: http://xx.xx.xx.xx/js/status.js"

After that I added in config.yaml username and password in the plant.xxxx area.

That results in "WARNING:omnik.datalogger:W 2023-01-29T13:54:22.617066 Request error: ('Connection aborted.', BadStatusLine('<html>\r<head>\r<Meta http-equiv=Content-Type content=text/html charset=gb2312>\r<title>web error</title>\rERROR:404 Not Found\r</head>\r</body>\r</html>'))"

I'll configure username and password for all 3 microinverters now and will let it run some days.

Because not at home next days, will report in a few days.

In Advance: Thank you for your work and your support.

**Comment by jbouwh** [comment]
Okay if they are new, then may be they are not supported to access locally. You could try the username password method, that should work to if your stats are supplied by Solarman.

JensHoRi ***@***.***> schreef op 29 januari 2023 23:13:47 CET:

**Comment by JensHoRi** [comment]
SOLARMAN works fine, but want to get rid of it. Saw somewhere else port 8899 is binded to AP-Mode in inverter, but that’s possible to change with some tricks. The port 8899 is open, already checked that. 

Give me some days.

**Comment by jbouwh** [comment]
Right, in that case may be you could turn on `debug` logging in your config to see what is happening:

The code where the connection is made can be found here:

https://github.com/jbouwh/omnikdatalogger/blob/4fad998b2ab6b5b07f4082a8bbb1fdd64cdc4a98/apps/omnikdatalogger/omnik/plugin_client/tcpclient.py

The message return should have minimal length of 99.

**Comment by jbouwh** [comment]
Known support is for WiFi modules with s/n 602xxxxxx and 604xxxxxx, those should support direct access thru port 8899.
This is about the S/N of the logger, not the inverter.

**Comment by JensHoRi** [comment]
my starting with 4, but maybe because from deye. I ll report

**Comment by JensHoRi** [comment]
Well, I was able to change the 8899 IP on my Deye Inverters (some tricky, but worked), Port 8899 ist available (could connect via telnet), but omnikdatalogger only throws errors:
Feb  2 15:06:09 alderaan omniklogger.py[3180038]: INFO:omnik.datalogger:I 2023-02-02T15:06:09.564585 Initializing: Trying to reach the inverter for plant xxxxxxxxxxx over port 8899.
Feb  2 15:06:49 alderaan omniklogger.py[3180038]: INFO:omnik.datalogger:I 2023-02-02T15:06:49.603882 Inverter is not reachable over port 8899, this is normal when it is dark. timed out

Hm, I'll try another solution, but thank you very much for your answers and all your development.
